lint-python/action.yml

```diff
@@ -0,0 +1,19 @@
+name: Lint Python
+description: 'Linting for Python projects'
+inputs:
+  exclude_trailing_comma_path:
+    required: false
+    description: 'Ignore trailing comma changes in given path'
+    default: ''
+
+runs:
+  using: "composite"
+  steps:
+    - run: find ${{ inputs.exclude_trailing_comma_path }} -name '*.py' | xargs poetry run add-trailing-comma --exit-zero-even-if-changed --py36-plus
+      if: ${{ inputs.exclude_trailing_comma_path }} != ''
+      shell: bash
+
+    - run: |
```

Comment:
I would specify a name for the step, because you execute two commands. E.g. `Run linter`. In the GitHub actions logfile you need to expand the command to what is really executed.

Otherwise you can put it to another `run` step.

Reply:
Thanks for the feedback. Sorry, I don't quite understand what you are suggesting. The logfile [here](https://github.com/moneymeets/django-insurance-api-v2/actions/runs/3224685636/jobs/5276087184), for example, looks good to me. Can you give an example where you would like to add the name? Thank you!

Reply:
I have split up the run steps as we have discussed, @marns93. 

As for leaving out names, after looking at 
![Screenshot 2022-10-11 at 10 51 52](https://user-images.githubusercontent.com/610633/195058508-910f0425-3e30-4649-b8d2-baca3c522d69.png)
I might still have preference to simply call the step "Run linter", but this is something we can decide in the pull requests that change the ci.yml workflows in each repository.